### PR TITLE
Member Profile for  Stephen Miller

### LINKED ISSUE #1776: [Community] Member Profile: Stephen Miller
**Current Behavior**

@smille02a has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @smille02a as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: `@smille02a`
- Twitter: `@smille02`
- LinkedIn: http://www.linkedin.com/in/swmps
- Link to profile picture: 
profile-1602695611518

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Signed-off-by: Anita-ihuman <charlesanita403@gmail.com>

**Description**
Added Member Profile for  Stephen Miller

This PR fixes #1776

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/members/stephen-miller /index.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (Tanuj22) [inline src/collections/members/stephen-miller /index.mdx:1]: There is a leading white space in the folder name. Remove that

File: src/collections/members/stephen-miller /index.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+name: Stephen Miller 
+position: Contributor
+image_path: ./stephen-miller.jpeg 
+github: Stephen Miller
```
Comment (Tanuj22) [inline src/collections/members/stephen-miller /index.mdx:5]:
```suggestion
github: smille02a
```

File: src/collections/members/stephen-miller /index.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+name: Stephen Miller 
+position: Contributor
+image_path: ./stephen-miller.jpeg 
+github: Stephen Miller 
+titHub: @smille02a
```
Comment (Tanuj22) [inline src/collections/members/stephen-miller /index.mdx:6]: Remove

File: src/collections/members/stephen-miller /index.mdx [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+twitter: @smille02
```
Comment (Tanuj22) [inline src/collections/members/stephen-miller /index.mdx:7]:
```suggestion
twitter: smille02
```

File: src/collections/members/stephen-miller /index.mdx [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+linkedIn: Stephen Miller
+location: Sure, Manhattan, NYC, NY, US
```
Comment (Tanuj22) [inline src/collections/members/stephen-miller /index.mdx:9]:
```suggestion
location: NYC, NY, USA
```

**Review by Tanuj22 — CHANGES_REQUESTED** [review]

File: src/collections/members/stephen-miller/index.mdx [hunk 6]
```diff
@@ -0,0 +1,13 @@
+---
+name: Stephen Miller 
+position: Contributor
+image_path: ./stephen-miller.jpeg 
+github: smille02a
+twitter: smille02
+linkedIn: Stephen Miller
```
Comment (Tanuj22) [inline src/collections/members/stephen-miller/index.mdx:7]:
```suggestion
linkedIn: swmps
```

**Comment by Tanuj22** [comment]
@Anita-ihuman the DCO is failing

**Comment by leecalcote** [comment]
@Anita-ihuman that's an issue with GitHub. I've flagged this DCO to pass for the current commits.

**Comment by l5io** [comment]
🚀 Preview for commit b6b69f4fb936c13b9e0c77bdcbb73e03ab3d7e6b at: https://607878c6a8875e793dc57e15--layer5ng.netlify.app

**Comment by Tanuj22** [comment]
@Anita-ihuman I ended up fixing the build error

**Review by Tanuj22 — APPROVED** [review]

**Pull request merged**